### PR TITLE
Allow the publisher to define the logStreamName per job

### PR DESCRIPTION
**Pull request by samdunne**

### CODE REVIEW

File: src/main/resources/jenkins/plugins/awslogspublisher/AWSLogsPublisher/config.jelly [hunk 1]
```diff
@@ -24,9 +24,9 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <!--
-    This jelly script is used for per-project configuration.
-
-    See global.jelly for a general discussion about jelly script.
-  -->
+  <f:section title="${%AWS CloudWatch Logs Publisher}">
+    <f:entry title="${%Log group name}" field="logStreamName">
```
Comment (elifarley) [inline src/main/resources/jenkins/plugins/awslogspublisher/AWSLogsPublisher/config.jelly]: It should read "%Log stream name".

File: src/main/java/jenkins/plugins/awslogspublisher/AWSLogsPublisherDescriptor.java [hunk 2]
```diff
@@ -23,9 +28,9 @@ protected AWSLogsPublisherDescriptor(Class<? extends Publisher> clazz) {
         super(clazz);
     }
 
-    /**
-     * This human readable name is used in the configuration screen.
-     */
+        /**
```
Comment (elifarley) [inline src/main/java/jenkins/plugins/awslogspublisher/AWSLogsPublisherDescriptor.java]: Lines 31 - 33: No need to change them.

**Review by elifarley — CHANGES_REQUESTED** [review]
Hi,
Please take a look at my comments on this review.

And thanks for your contribution!

**Comment by samdunne** [comment]
Can we possibly release this change?

**Pull request merged**